implementations/rust/ockam/ockam_command/src/tcp/inlet/create.rs

```diff
@@ -153,6 +153,14 @@ pub struct CreateCommand {
     /// Requires `ockam-tls-certificate` credential attribute.
     #[arg(long, value_name = "ROUTE", hide = true)]
     pub tls_certificate_provider: Option<MultiAddr>,
+
+    /// Skip Portal handshake for lower latency, but also lower throughput
```

Comment:
It would be helpful for the user to know that he'll need to specify `skip_handshake` in the outlet as well (and the reverse as well)